### PR TITLE
Add updatedependencies handler for corefx release branch.

### PR DESCRIPTION
**Pull request by Priya91**
cc @eerhardt @joshfree

### CODE REVIEW

File: Maestro/subscriptions.json [hunk 1]
```diff
@@ -1,11 +1,17 @@
 {
   "actions": {
-    // A build definition capable of running any .ps1 script in the CoreFX repo
+    // A build definition capable of running any .ps1 script in the CoreFX repo master branch
     "corefx-general": {
       "vsoInstance": "devdiv.visualstudio.com",
       "vsoProject": "DevDiv",
       "buildDefinitionId": 1084
     },
+    // A build definition capable of running any .ps1 script in the CoreFX repo release branch
```
Comment (eerhardt) [inline Maestro/subscriptions.json]: can you say release/1.0.0 branch?  To make it more specific.

Like I said offline, with a new Maestro feature, you will be able to specify the branch without having to create a new VSO build definition.  I'll try to get this today or tomorrow.

File: Maestro/subscriptions.json [hunk 2]
```diff
@@ -36,6 +42,24 @@
       ]
     },
     {
+      "path": "https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/release/1.0.0/Latest.txt",
+      "handlers": [
+        // This handler will bring the Latest CoreFX release build into CorefX release
```
Comment (eerhardt) [inline Maestro/subscriptions.json]: Same comment - `release/1.0.0`.

**Comment by eerhardt** [comment]
minor nits on the comments.  When fixed, I'll merge.

**Comment by Priya91** [comment]
@eerhardt made the changes..

**Pull request merged**